.github/workflows/codespell.yaml

```diff
@@ -0,0 +1,23 @@
+name: Codespell
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
+  pull_request:
+    branches:
+      - "master"
+jobs:
+  codespell:
+    name: codespell
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: codespell
+        uses: codespell-project/actions-codespell@master
+        with:
+          skip: .git,*.png,*.jpg
```

Comment:
Please also add *.svg which is used for Rook's logo.

Reply:
Well, they don't trigger errors right now, but sure :+1: